redis/RedisArray.php

```diff
@@ -0,0 +1,213 @@
+<?php
+
+/**
+ * Helper autocomplete for php redis extension
+ *
+ * @mixin \Redis
+ *
+ * @author Avtandil Kikabidze <akalongman@gmail.com>
```

Comment:
Could you please remove the `@author` and a link to your personal info? No one does this and this information doesn't look suitable here.

Reply:
No one does? :)
![Screenshot from 2022-02-18 18-34-20](https://user-images.githubusercontent.com/423050/154702813-c5fc7175-fcd1-422e-a73e-f790bc536ed0.png)
Ok, removed